backend/candidate_comparison_tool/router.py

```diff
@@ -0,0 +1,52 @@
+from fastapi import APIRouter, HTTPException
+from fastapi.concurrency import run_in_threadpool
+from pydantic import BaseModel
+
+from .exceptions import GeminiError
+from .service import get_gemini_score
```

Comment:
Prefer absolute imports vs. releative imports.
```suggestion
from backend.candidate_comparsion_tool.exceptions import GeminiError
from backend.candidate_comparsion_tool.service import get_gemini_score
```